app/controllers/collection_drafts_controller.rb

```diff
@@ -192,12 +192,20 @@ def validate_metadata
     end
     JSON::Validator.register_format_validator('date-time', date_time_format_proc)
 
-    errors = Array.wrap(JSON::Validator.fully_validate(@json_schema, get_resource.draft))
+    errors = Array.wrap(JSON::Validator.fully_validate(@json_schema, fix_metadata_fields(get_resource.draft)))
     errors = validate_paired_fields(errors, get_resource.draft)
     errors_before_generate = validate_picklists(errors, get_resource.draft)
     errors_before_generate.map { |error| generate_show_errors(error) }.reject(&:nil?).flatten
   end
 
+  def fix_metadata_fields(meta)
```

Comment:
I think this fix should be renamed to be more explicit about what it is fixing. Also, I think this should be moved to the model, that's where other fixes like this are.